com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -123,12 +120,15 @@ protected override void Dispose(bool disposing)
 
         protected override void Render(ScriptableRenderContext renderContext, Camera[] cameras)
         {
-#if UNITY_2020_2_OR_NEWER
+            // TODO: Would be better to add Profiling name hooks into RenderPipelineManager.
```

Comment:
Because this will need changes outside of this repository. Also the enum is tracked by our perf framework, and I don't know yet if we can have both. Plan is to look at that after this gets merged.